proto/logging_config.d.ts

```diff
@@ -0,0 +1,5376 @@
+import * as $protobuf from "protobufjs";
```

Comment:
I am confused about this file being part of the PR :) Is this needed here?  We already have `logging.d.ts` in the proto dir - what's this?